ffc/billing/process_billing.py

```diff
@@ -184,25 +209,27 @@ async def evaluate_journal_status(self, journal_external_id) -> dict[str, Any] |
         """
         journal = await self.mpt_client.get_journal(self.authorization_id, journal_external_id)
         if not journal:
-            self.logger.error(f"No journal found for external ID: {journal_external_id}")
+            error_msg = f"No journal found for external ID: {journal_external_id}"
+            self.logger.info(error_msg)
             return None
-
         journal_id = journal["id"]
         journal_status = journal["status"]
-        if journal_status in (VALIDATED, DRAFT):
-            self.logger.info(f"Already found journal: {journal_id} with status {journal_status}")
+        if journal_status in VALID_STATUS:
+            msg = f"Already found journal: {journal_id} with status {journal_status}"
+            self.logger.info(msg)
             return journal
         else:
-            self.logger.warning(f"Found the journal {journal_id} with status {journal_status}")
-            raise JournalStatusError()
+            error_msg = f"Found the journal {journal_id} with status {journal_status}"
+            self.logger.error(error_msg)
+            raise JournalStatusError(error_msg, journal_id)
 
-    async def process(self) -> AuthorizationProcessResult | None:
+    async def process(self) -> ProcessResultInfo | None:
```

Comment:
This method must always return a ProcessResultInfo class; None is not an option.
The process can succeed or fail. The ProcessResultInfo has statuses that tell if the process succeeded or failed. What does `None` mean?